#!/bin/sh /usr/share/dpatch/dpatch-run
## 335-magic-add-freemind.dpatch by Jamie Thompson <debian-bugs@jamie-thompson.co.uk>
##
## DP: New magic for Freemind (Closes: #472385).

@DPATCH@

diff -Naurp file.orig/magic/Magdir/wordprocessors file/magic/Magdir/wordprocessors
--- file.orig/magic/Magdir/wordprocessors	2008-05-04 14:12:49.000000000 +0000
+++ file/magic/Magdir/wordprocessors	2008-05-04 14:18:38.000000000 +0000
@@ -148,3 +148,8 @@
 0	string		DOC
 >43	byte		0x16	Just System Word Processor Ichitaro v6
 !:mime	application/x-ichitaro6
+
+# Type: Freemind mindmap documents
+# From: Jamie Thompson <debian-bugs@jamie-thompson.co.uk>
+0	string/cB	\<map\ version	Freemind document
+!:mime	application/x-freemind
